package.json

```diff
@@ -2,6 +2,7 @@
   "name": "gulp-append-prepend",
   "version": "1.0.8",
```

Comment:
```suggestion
  "version": "1.0.9",
```

Reply:
Wasn't sure if the changes even warranted a patch version bump, so left as-is initially - commited!